tests/statecheck/consensus_test.go

```diff
@@ -137,11 +137,9 @@ func TestGenesisFull(t *testing.T) {
 
 	ctx := context.Background()
 
-	factory, err := newSourceClientFactory()
-	require.Nil(t, err)
-
-	oasisClient, err := factory.Consensus()
+	conn, err := newSdkConnection(ctx)
 	require.Nil(t, err)
+	oasisClient := conn.Consensus()
```

Comment:
:information_source:  the factory would do signature.SetChainContext, where the SDK connection doesn't. we have a few `.Open()`, but luckily they're on registry things that don't use the chain context

Reply:
thanks for the tip; i was making the same mistake in the api-tests